lib/middlewares/requestLog.js

```diff
@@ -2,6 +2,11 @@ module.exports = function(req, res, next) {
 
   req._startTime = new Date
 
+  let oldWrite = res.write
```

Comment:
could we name it `originalWrite` ad `originalEnd` instead, and how about `res.send` lol